docs/src/tutorial.md

```diff
@@ -1 +1,63 @@
-# LDLFactorizations.jl Tutorial
\ No newline at end of file
+# LDLFactorizations.jl Tutorial
+
+## A basic example
+
+```julia
+n = 10
+A0 = rand(n,n)
+A = A0 + A0' # A is symmetric
+b = rand(n)
+```
+
+We solve the system `A x = b` using LDLFactorizations.jl :
+
+```julia 
+using LDLFactorizations, LinearAlgebra
+Au = triu(A) # get upper triangle
+LDL = ldl(Symmetric(Au, :U))
+x = LDL \ b
+```
+
+## A more performance-focused example
+
+We build a problem with sparse arrays.
+
+```julia
+using SparseArrays
+n = 100
+A0 = sprand(Float64, n, n, 0.1)
+A = A0 * A0' + I # A is symmetric
+b = rand(n)
+```
+
+Now if we want to use the factorization to solve multiple systems that have 
+the same sparsity pattern as A, we only have to use `ldl_analyze` once.
+
+```julia 
+Au = triu(A) # get upper triangle
```

Comment:
Why? This will not work if I remove `triu`. The user either has to do: `ldl(A)` or `ldl(Symmetric(triu(A), :U))`. Or do you want me to call `triu` on `Au` after?

Reply:
That’s true. I just wanted to avoid writing `Symmetric(...)` all the time. Can you set `Au = Symmetric(triu(A), :U)`?